### DISCUSSION TITLE
use IOS shortcut automation to achieve similar effects on iPhone

### DISCUSSION THREAD

**Discussion by chenbei1688** [t1m0thyj/WinDynamicDesktop#313] [Ideas]
Hello, this project is great, thank you very much!

I would like to ask if you can use IOS shortcut automation to achieve similar effects on iPhone?

Look forward to your discussion!

**Comment by t1m0thyj** [comment]
Hi, I don't have any plans to create an iOS app, but you can find some dynamic wallpapers for iOS here: https://www.jetsoncreative.com/24hourios
